subset/switches/src/main/java/switchtest/SwitchInterrogator.java

```diff
@@ -445,6 +439,49 @@ private String cleanShowRunData(String data) {
     return data;
   }
 
+  private void validateTests() {
+    try {
+      System.out.println("Tests validating....");
+      // switch.port.link test passes is "Link is UP" and dropped packets being equal to 0.
+      if (show_interface_data[1].equals("UP") && Integer.parseInt(show_interface_data[12]) == 0) {
```

Comment:
Random constants like this (12) are really bad -- they should be a named constant that's sourced somewhere near where it has some meaning. E.g. just "DROPPED_PACKETS_INDEX" -- and then somewhere have DROPPED_PACKETS_INDEX = 12.  IntelliJ at least has easy helpers to do this (extract constant).

If you do that, then you don't need the comment because it's obvious from the code:

Reply:
it looks like the time has come to implement the hashmap to store data efficiently using keys